### ISSUE TITLE
[Bug] Shell SearchHandler Results view on iOS not dismissing

### ISSUE THREAD

**Issue by Shaggy13spe** [xamarin/Xamarin.Forms#11000]
### Description
When entering terms in to the SearchHandler, the "suggested" results view displays, correctly. However, upon hitting "Search/Enter" in the keyboard, the results view does not disappear and blocks the views beneath. You have to actually clear out the searchbox before being able to see the view below.

### Basic Information

- Version with issue: Xamarin.Forms 4.6.0.847
- Last known good version: 
- IDE: VS for Mac 2019
- Platform Target Frameworks: 
  - iOS:  13.5

- Affected Devices:
iOS  devices

**Comment by hartez** [comment]
@Shaggy13spe Is this happening in a Shell app or a classic app?

**Comment by Shaggy13spe** [comment]
Shell application.

**Comment by samhouts** [comment]
Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by Shaggy13spe** [comment]
@samhouts Sorry, it's been a very hectic week. We go live with our application on July 1, and there are several backend processes that are still being finished. This seems to be the only defect we see in the client app, but the backend stuff has taken precedence over it. I will try to put together a small application to demonstrate as well, but I've recorded the problem with our app and attached here. In it, I search for "filter" and hit the search button, which used to then dismiss the "search suggestions" list and clear the search field and show the results. Manually clearing the field get's rid of the suggestion list, and after unfocusing the search bar, I have access to the search results.

[RPReplay_Final1592680615.MP4.zip](https://github.com/xamarin/Xamarin.Forms/files/4808450/RPReplay_Final1592680615.MP4.zip)

**Comment by NerdArray** [comment]
I'm having the same issue.  I made a quick example app from the Flyout template.  In Android it's fine, the suggested results disappear when you press enter.  On iPhone it does not.  I have tried to clear the Query property and tried overriding OnQueryConfirmed() in my search handler and calling Unfocus().  I also tried data binding to ShowResults and setting it False when when Command was called.

[App1.zip](https://github.com/xamarin/Xamarin.Forms/files/5762541/App1.zip)

**Comment by upsilondynamics** [comment]
Is this issue resolved?

**Comment by Shaggy13spe** [comment]
No, and according to the activity history, it seems it isn't going to be any time soon
